agent/command/github_generate_token.go

```diff
@@ -130,3 +136,23 @@ func (r *githubGenerateToken) Execute(ctx context.Context, comm client.Communica
 
 	return nil
 }
+
+func permissionsToString(permissions *github.InstallationPermissions) string {
```

Comment:
although this is a private function so doesn't require a comment, it might be nice to have one to summarize what the output looks like